docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '2'
+services:
+  web:
+    image: "myimage"
+    ports:
+      - 8080:8080
+    depends_on:
+      - db
+    environment:
+      - POSTGRES_ENV_PASSWORD=POSTGRES_PASSWORD
+      - POSTGRES_ENV_USER=POSTGRES_USER
+      - POSTGRES_URL=jdbc:postgresql://db:5432/postgres
```

Comment:
@blackavgn externalize this configuration

Reply:
As i understand task, everything in my POSTGRES_URL not variable(database name). So i can define it just as String.Moreover in my meaning it's not secure data. So i need to provide my user access to change database name?Is it useful?